shared/src/business/utilities/createSignedDocumentsForInspection.js

```diff
@@ -0,0 +1,47 @@
+// To directly run via node:
+//   npm install -g  @babel/core @babel/node
+//   npx babel-node shared/src/business/utilities/createSignedDocumentsForInspection.js
+
+const fs = require('fs');
+const path = require('path');
+const {
+  applicationContext: applicationContext,
+} = require('../../../../web-client/src/applicationContext');
+const {
+  generateSignedDocumentInteractor,
+} = require('../useCases/generateSignedDocumentInteractor');
+
+// see this for reference: shared/src/business/utilities/documentGenerators.test.js
```

Comment:
Could we add a bit more about the purpose of this script? Use this to do X?

Reply:
You bet!

Reply:
Updated!